packages/x-tailwindcss/src/x-tailwind-plugin/components/input/default.ts

```diff
@@ -0,0 +1,37 @@
+import { TailwindHelpers } from '../../../types';
+import { inputColors } from './colors';
+import { inputSizes } from './sizes';
+
+/**
+ * Returns the default styles for component `input`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function inputDefault(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    fontFamily: theme('fontFamily.main'),
+    borderWidth: theme('borderWidth.1'),
+    cursor: 'pointer',
+    fontWeight: theme('fontWeight.regular'),
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    gap: theme('spacing.8'),
```

Comment:
question: why do we need these styles? are we planning to apply the input classes to other elements?

Reply:
It is specified on the designs that we should do ellipsis if necessary

Reply:
Oh, I think I broke Chrome 😅. Yesterday this wasn't working for me, now it does. AFAIK this only works when you don't have the focus in the input. If I type a long query and I lose the focus by moving it into another screen it doesn't add the ellipsis.
![image](https://user-images.githubusercontent.com/68222542/208844868-a7e6d589-92bd-48bf-863a-99c5a2aebcb8.png)

 If I lose the focus by clicking on the page it does add it:
![image](https://user-images.githubusercontent.com/68222542/208844902-b8915641-e043-40a9-82b5-ef41d5d974f8.png)

However, I still don't get the `gap` style.  The input does not have a flex or grid display.

Reply:
![image](https://user-images.githubusercontent.com/68222542/208845990-98d67d07-a4fd-450a-9925-b0094ab005fe.png)
Seems that `white-space` `overflow` and `gap` are not needed for that effect.

Reply:
Are they needed for the animated placeholder? Also this styles are conflicting with the base input and input group one, so we have to be careful when enabling them.

Reply:
I think the `gap` is not needed as the input would not have multiple children.  The `white-space` & `overflow` are not needed because it is an `input` with that implicit behaviour. So we can remove them

Surely we must do some more work to make this work with the animated placeholder. That's one of the reasons I preferred to use the proper placeholder to be animated instead of another element.

I think we have to use the `input group` to use the animated placeholder, so I would leave that case for that task.


@javieri-empathy I don't understand what you mean by "conflicting with the base input and input group"?
do you mean the old DS? those will be deprecated once we use the new ones.

Reply:
That was it, just wanted to confirm that this was the intention of this PR. Thanks.

Reply:
Yes you are right, we can remove those properties. The thing is that you can not have an ellipsis on the input if you have the focus, I think that this is intended to read everything when you have the focus (at least in In Webkit-based browsers). I don't know if there is a hack to add an ellipsis always, but I think that it makes no sense.

https://jsfiddle.net/simurai/TjGyz/